components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -461,6 +467,64 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+    const [isFeedbackFormVisible, setFeedbackFormVisible] = useState<boolean>(false);
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert
```

Comment:
**suggestion**: One comment that was brought up in the latest IDE / Design Check-in was to move this alert above the form to keep the placement of the alert the same as the rest of the form errors or experiment with placing in between the form inputs and the form submit button. This will also avoid splitting the form submit button and the auto-start checkbox which is closely related. WDYT @mustard-mh?

| CURRENT | Above form | Inside form |
|-|-|-|
| <img width="1512" alt="Screenshot 2023-07-05 at 15 31 16" src="https://github.com/gitpod-io/gitpod/assets/120486/f1a69932-e6e1-4533-b3cc-7ae9b9a8b732"> | <img width="1512" alt="Screenshot 2023-07-05 at 15 31 43" src="https://github.com/gitpod-io/gitpod/assets/120486/c9ee8355-a881-4982-b17a-07fd2b3e5cbb"> | <img width="1512" alt="Screenshot 2023-07-05 at 15 32 03" src="https://github.com/gitpod-io/gitpod/assets/120486/6bd0404a-c09a-43ff-a9bd-5806d772f6c7"> |

Reply:
I put this view below `continue` button to avoid user need to move their mouse more to create workspace (option location will change) 🤔